creme/sketch/bricks.py

```diff
@@ -59,14 +60,14 @@ class BarChartBrick(ChartBrick):
     """
     template_name = "sketch/bricks/bar-chart.html"
 
-    # Draw limit lines on the Y axis
-    limits = []
+    # Show limits along the Y axis : list of values drawn as horizontal lines
+    limits: Sequence[Union[int, float]] = ()
```

Comment:
I gave you the code I wanted `int | float`... Can you please read my reviews please.